persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/PersistentEntity.scala

```diff
@@ -17,9 +18,11 @@ object PersistentEntity {
    *
    * `akka.Done` can optionally be used as a "standard" acknowledgment message.
    *
-   * Type parameter `R` is the type of the reply message.
+   * @tparam R the type of the reply message
    */
-  trait ReplyType[-R]
+  trait ReplyType[R] {
+    type ReplyType = R
```

Comment:
This was exactly what I tried, but then got stuck because of variance, isn't having R invariant this a problem if the response is an ADT?